tensorflow_io/core/plugins/az/az_file_system.cc

```diff
@@ -31,6 +31,12 @@ limitations under the License.
 #include "storage_account.h"
 #include "storage_credential.h"
 #include "storage_errno.h"
+// TODO: Restore logging.h
+#define TF_Log(...)
+#define TF_VLog(...)
```

Comment:
@yongtang the `TF_VLog` macro isn't being used here, is this just to bring in the existing logging macro's ?

Reply:
@kvignesh1420 The `TF_VLog` is part of the C logging API in tensorflow (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/c/logging.h#L34-L36). Currently it is defined as empty as we need https://github.com/tensorflow/tensorflow/pull/44993 . Once the PR is merged those defines here can be safely removed.

Reply:
okay!